platform/security/core/security-core-impl/src/test/groovy/ddf/security/samlp/impl/LogoutMessageSpecTest.groovy

```diff
@@ -83,7 +83,7 @@ class LogoutMessageSpecTest extends Specification {
 
     def "build logout request with invalid info"() {
         when:
-        LogoutRequest logoutRequest = logoutMessage.buildLogoutRequest(nameId, issuerId, id)
+        LogoutRequest logoutRequest = logoutMessage.buildLogoutRequest(nameId, issuerId, id, SESSION_INDEXES)
```

Comment:
❓ If session indices cannot be `null`, we need to add tests for that here.

Reply:
Added this line to test for null:
`"xxx"  | "xxx"    | "xxx" | null`